libobs/util/windows/win-version.h

```diff
@@ -48,6 +48,7 @@ static inline int win_version_compare(const struct win_version_info *dst,
 }
 
 EXPORT bool is_64_bit_windows(void);
+EXPORT bool is_arm64_windows(void);
```

Comment:
Is this function used anywhere outside platform-windows.c or intended to be used? Or could the definition here get removed?

Reply:
It could get used in something like Python scripting #3360

Reply:
Oh yeah that makes sense.
Now where did GH put the *Mark Resolved* button? 🤔